### PR TITLE
Feat: new layout with main menu (and Style 10)

### PR DESCRIPTION
**Pull request by falric**
> [!NOTE]  
> There’s still development in other apps that might not have made into this branch.
>
> At this point actual applications might give you better guidance. See [KTH/search-web](https://github.com/KTH/search-web) for example.

## Summary

- A new take on restyling `node-web` with KTH Style 10
- Reference [KTH Style 10 #107](https://github.com/KTH/node-web/pull/107) for notes
- Relates to [Skapa app för nya grafiska profilen](https://trello.com/c/tO2UmHAO/2993-skapa-app-f%C3%B6r-nya-grafiska-profilen) and [Vänstermeny i Node-appar](https://trello.com/c/IDNBNJGT/3168-v%C3%A4nstermeny-i-node-appar)

### CODE REVIEW

File: i18n/messages.se.js [hunk 1]
```diff
@@ -41,5 +41,11 @@ module.exports = {
     template_silent_login_page_heading: 'Sida med tyst inloggning',
     template_not_found_page_heading: 'Sidan hittades inte ',
     template_back_link: 'Tillbaka till start-sidan',
+
+    language_link_lang_en: 'English',
+    language_link_not_translated: 'This page isn’t translated',
+    language_link_button_close: 'Stäng',
```
Comment (nsandstrom) [inline i18n/messages.se.js:47]: Should the button label be swedish?
We are on the swedish page, but the rest of this dialog is in english.

**Comment by nsandstrom** [comment]
image

Should the main content not be a bit more centered in window?
(this is how it looks on my machine at least)

**Comment by ahrsjo** [comment]
Does this relate to this card? https://trello.com/c/IRuayimS/3140-node-appar-kth-style-10-migrering-fas-1

**Comment by falric** [comment]
> image
> Should the main content not be a bit more centered in window? (this is how it looks on my machine at least)

Yep. That needs to be fixed. The `kth-content` should set `max-width: 82rem`.

**Comment by falric** [comment]
> Does this relate to this card? https://trello.com/c/IRuayimS/3140-node-appar-kth-style-10-migrering-fas-1

More [Skapa app för nya grafiska profilen](https://trello.com/c/tO2UmHAO/2993-skapa-app-f%C3%B6r-nya-grafiska-profilen). I added a description to the pull request.

**Comment by ahrsjo** [comment]
Should we fix the `max-width`, rebase the branch and merge? 😊

**Comment by falric** [comment]
> image
> Should the main content not be a bit more centered in window? (this is how it looks on my machine at least)

Fixed!

**Comment by falric** [comment]
> Should we fix the `max-width`, rebase the branch and merge? 😊

Yep!

**Pull request merged**